src/docs/asciidoc/index.adoc

```diff
@@ -243,6 +243,12 @@ include::{snippets}/PortfolioControllerTest/mustBeAbleToGetBlockchainTransaction
 .Response
 include::{snippets}/PortfolioControllerTest/mustBeAbleToGetBlockchainTransactions/http-response.adoc[]
 
+=== Get all project transactions
+.Request
```

Comment:
Add a comment: "Only organization members can see a list of project transactions."